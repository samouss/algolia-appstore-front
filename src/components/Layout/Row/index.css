.Row {
  width: 100%;
  background-color: #ffffff;
  border: 1px solid #f2f2f2;
  margin-top: 1vh;

  &:first-child {
    margin-top: 0;
  }
}
