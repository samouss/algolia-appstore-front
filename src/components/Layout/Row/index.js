import React from 'react';
import PropTypes from 'prop-types';
import './index.css';

const Row = ({ children }) => (
  <div styleName="Row">
    {children}
  </div>
);

Row.propTypes = {
  children: PropTypes.node.isRequired,
};

export default Row;
