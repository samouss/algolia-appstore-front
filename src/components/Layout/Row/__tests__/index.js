import React from 'react';
import { shallow } from 'enzyme';
import Row from '../index';

describe('<Row />', () => {
  const defaultProps = {};

  it('expect to render', () => {
    const props = {
      ...defaultProps,
    };

    const component = shallow(
      <Row
        {...props}
      >
        Content
      </Row>,
    );

    expect(component).toMatchSnapshot();
  });
});
