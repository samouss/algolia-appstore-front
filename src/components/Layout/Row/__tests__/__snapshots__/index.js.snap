// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Row /> expect to render 1`] = `
<div
  styleName="Row"
>
  Content
</div>
`;
